src/components/chart/tests/LineSeries.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+
+import {LineSeries} from '../LineSeries';
+import {XYChartContextMock} from './XYChartContextMock';
+
+describe('<LineSeries />', () => {
+    it('should not throw', () => {
+        expect(() => {
+            shallow(<LineSeries />);
+            shallow(<LineSeries interpolate='monotone' strokeWith={2} />);
+        }).not.toThrow();
+    });
+
+    it('should render a <g>', () => {
+        const component = shallow(<LineSeries />);
+        expect(component.find('g').exists()).toBe(true);
+    });
+
+    it('should render a path for every serie', () => {
+        spyOn(React, 'useContext').and.returnValue(XYChartContextMock);
+
+        const {series} = XYChartContextMock;
+        const component = shallow(<LineSeries />);

```

Comment:
Instead of spying on useContext, it is probably cleaner to pass down the context directly to shallow:
```tsx
const component = shallow(<LineSeries />, {context: XYChartContextMock});
```
Same comment goes for the other places where you spy on `useContext`

Reply:
I haven't been able to make it work. It might be working with a mount but I would wait for better support from enzyme